src/components/learner-credit-management/assignments-status-chips/NotifyingLearner.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Chip, useToggle } from '@edx/paragon';
+import { Send } from '@edx/paragon/icons';
+import BaseModalPopup from './BaseModalPopup';
+
+const NotifyingLearner = ({ learnerEmail }) => {
+  const [isOpen, open, close] = useToggle(false);
+  const [target, setTarget] = useState(null);
+
+  return (
+    <>
+      <Chip
+        ref={setTarget}
+        iconBefore={Send}
+        onClick={open}
+        onKeyPress={open}
+        tabIndex={0}
+      >
+        Notifying learner
+      </Chip>
+      <BaseModalPopup
+        positionRef={target}
+        isOpen={isOpen}
+        onClose={close}
+      >
+        <BaseModalPopup.Heading icon={Send} iconClassName="text-gray-300">
+          Notifying {learnerEmail ?? 'learner'}
+        </BaseModalPopup.Heading>
+        <BaseModalPopup.Content>
+          <p>
+            Our system is busy emailing {learnerEmail ?? 'the learner'}! Refresh in a few minutes to
+            confirm the assignment notification was successful.
+          </p>
+        </BaseModalPopup.Content>
+      </BaseModalPopup>
+    </>
+  );
+};
+
+NotifyingLearner.propTypes = {
+  learnerEmail: PropTypes.string,
```

Comment:
nit: isRequired?  Assignment responses should always have an associated email.

Reply:
With the GDPR/PII pipeline eventually getting integrated for assignments (https://2u-internal.atlassian.net/browse/ENT-7557), I believe there will eventually no longer be a guarantee that `learner_email` will always be present if/when PII data gets scrubbed for assignments. Being proactive and handling those situations in the UI now similar to how we have conditional logic for PII scrubbing in the "Spent" table powered by the analytics API today (i.e., the table cell says "Email hidden" with a tooltip explaining why).

Reply:
ah good point